cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt

```diff
@@ -130,18 +165,24 @@ script "js_filter_resources", type: "javascript" do
         }
       }
     })
+    result = _.sortBy(result, 'location');
```

Comment:
should be sorting by subscription name first

Reply:
Updated.